contrib/basicauth/nginx/registry.conf

```diff
@@ -0,0 +1,17 @@
+# Docker registry proxy for api version 2
+
+upstream docker-registry-v2 {
+  server registryv2:5000;
+}
+
+
+server {
+  listen 5441;
+  server_name localhost;
+  ssl on;
```

Comment:
@noxiouz No worries. Unfortunately on this repo, I can't set labels, so I'm unable to label this appropriately.  It wasn't quite read for doc review but no matter.  The code itself is based on what Derek has already written in https://github.com/docker/distribution/tree/master/contrib/docker-integration  which is a really cool contribution.